e2e/404.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto('/nonexistent-url');
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
```

Comment:
```suggestion
test.beforeEach(async ({ page }) => {
  await page.goto('/nonexistent-url');
});

test.afterEach(async ({ page }) => {
  await page.close();
});
```

Ideally we should run tests in isolation.